File: Beginner/Exer. 1005/main.c
#include <stdio.h>
#define weightA 3.5
#define weightB 7.5

int main(int argc, char *argv[]) {
    double A, B;
    
    scanf("%lf", &A);
    scanf("%lf", &B);
    
    printf("MEDIA = %.5lf\n", ((A * weightA + B * weightB) / (weightA + weightB)));
    return 0;
}

File: Beginner/Exer. 1005/main.java
import java.util.Scanner;

class Main {
    public static final double weightA = 3.5;
    public static final double weightB = 7.5;

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double A, B;

        A = scanner.nextDouble();
        B = scanner.nextDouble();
        scanner.close();

        System.out.printf("MEDIA = %.5f\n", ((A * weightA + B * weightB) / (weightA + weightB)));
    }    
}